sirius-valley/lerni-mobile
# [LER-134] Trivia history modal

On this PR: 
.- Trivia history modal.

To test this feature: 
1. first of all you need to have the trivia finished with 'triviaMathId 0'
2. go to /(tabs)/trivia/index.tsx and uncomment lines 30 to 37.
3. reload the app.
4. go to trivia tab, trivia history modal should appear on first page
